WordPress/Classes/ViewRelated/Post/PostEditor+BlogPicker.swift

```diff
@@ -1,102 +0,0 @@
-                return
-            }
-            self.recreatePostRevision(in: blog)
-            self.mediaLibraryDataSource = MediaLibraryPickerDataSource(post: self.post)
-        }
-
-        let dismissHandler: BlogSelectorDismissHandler = {
-            self.dismiss(animated: true)
-        }
-
-        // Setup Picker
-        let selectorViewController = BlogSelectorViewController(selectedBlogObjectID: post.blog.objectID,
-                                                                successHandler: successHandler,
-                                                                dismissHandler: dismissHandler)
-        // Note:
-        // On iPad Devices, we'll disable the Picker's SearchController's "Autohide Navbar Feature", since
-        // upon dismissal, it may force the NavigationBar to show up, even when it was initially hidden.
-        selectorViewController.displaysNavigationBarWhenSearching = WPDeviceIdentification.isiPad()
-        if postIsReblogged {
-            selectorViewController.displaysOnlyDefaultAccountSites = true
-        }
-
-        // Setup Navigation
-        let navigationController = AdaptiveNavigationController(rootViewController: selectorViewController)
-        navigationController.configurePopoverPresentationStyle(from: sourceView)
-
-        // Done!
-        present(navigationController, animated: true)
-    }
-
-    func displaySwitchSiteAlert() {
-        let alert = UIAlertController(title: SwitchSiteAlert.title, message: SwitchSiteAlert.message, preferredStyle: .alert)
-
-        alert.addDefaultActionWithTitle(SwitchSiteAlert.acceptTitle) { _ in
-            self.displayBlogSelector()
-        }
-
-        alert.addCancelActionWithTitle(SwitchSiteAlert.cancelTitle)
-
-        present(alert, animated: true)
-    }
-
-    // TODO: Rip this and put it into PostService, as well
-    func recreatePostRevision(in blog: Blog) {
-        let shouldCreatePage = post is Page
-        let postService = PostService(managedObjectContext: mainContext)
-        let newPost = shouldCreatePage ? postService.createDraftPage(for: blog) : postService.createDraftPost(for: blog)
-        // if it's a reblog, use the existing content and don't strip the image
-        newPost.content = postIsReblogged ? post.content : contentByStrippingMediaAttachments()
-        newPost.postTitle = post.postTitle
-        newPost.password = post.password
-        newPost.dateCreated = post.dateCreated
-        newPost.dateModified = post.dateModified
-        newPost.status = post.status
-
-        if let source = post as? Post, let target = newPost as? Post {
-            target.tags = source.tags
-        }
-
-        discardChanges()
-        post = newPost
-        createRevisionOfPost()
-        RecentSitesService().touch(blog: blog)
-
-        // TODO: Add this snippet, if needed, once we've relocated this helper to PostService
-        //[self syncOptionsIfNecessaryForBlog:blog afterBlogChanged:YES];
```

Comment:
How about removing some of the methods, classes here? (`contentByStrippingMediaAttachments, discardChanges, createRevisionOfPost, RecentSitesService`, etc.)

Reply:
Removed `contentByStrippingMediaAttachments` at 5744f6a, rest are being used at other places like `editor + publish` etc